systemtest/tests/integration/test.sh

```diff
@@ -13,8 +13,10 @@ cd ../../../
 # This is for ad-hoc and compose testing.
 rpm -q insights-client || ./systemtest/guest-setup.sh
 
-dnf --setopt install_weak_deps=False install -y \
-  podman git-core python3-pip python3-pytest logrotate bzip2 zip
+  dnf --setopt install_weak_deps=False install -y \
+    podman git-core python3-pip python3-pytest logrotate bzip2 zip \
+    scap-security-guide openscap-scanner openscap
+fi
```

Comment:
The trailing `fi` doesn't have its `if` -- this branch isn't testing for bootc state. Drop the `fi` and decrease the indentation by two characters to the left again, we only need the openscap packages.